### ISSUE TITLE
Make the SleepTimerScreen screen reader accessible

### ISSUE THREAD

**Issue by mitchdowney** [podverse/podverse-rn#731]
I don't know how to set the accessibilityHint on the picker component. We need to announce which picker component is focused (hour, minute, seconds).

**Comment by mitchdowney** [comment]
I replaced the picker-based SleepTimer with the alternate experience of buttons (30 mins, 45mins, 1hr, 2hr, etc) when a screen reader is enabled. This should be more accessible.
